state:

  # Upper and lower bounds for experimental space (meters). [x, y, z]
  lower: [-2.94, -1.04, 0] #-2.98
  upper: [0.72, 2.62, 2.0] #0.77

pred:

  # Dimensions of the prediction map
  sim_height: 26
  sim_width: 26

  # Start position of human and list of potential goal locations
  sim_start: [0, 0] # [7, 25]
  sim_goals: [[13,25], [25,13]] #[[25, 25], [25, 7]] 

  # Resolution, ex. (real meters)/(sim dim-1) (m/cell)
  resolution: 0.1464 

  # Num timesteps into the future to predict
  fwd_tsteps: 10

  # Time between each step (in seconds?)
  fwd_deltat: 1

  # Initial rationality coefficient 
  init_beta: 0.5

  # Human's height (in meters)
  human_height: 1.67

  # Probability threshold 
  prob_thresh: 0.1
